packages/terra-embedded-content-consumer/src/terra-dev-site/test/embedded-content-consumer/consumers/Consumer.module.scss

```diff
@@ -0,0 +1,6 @@
+:local {
+  iframe {
```

Comment:
I believe this is applying styles to all `iframe` elements, should be changed to `.iframe`.

Reply:
I'll scope this